res/values-ar/strings.xml

```diff
@@ -1418,7 +1418,7 @@ Commands:
         <item>"صغير"</item>
         <item>"عادي"</item>
         <item>"كبير"</item>
-        <item>"ضخم"</item>
+        <item>"كبير جدا"</item>
```

Comment:
Also the same problem here, the old translation is the correct one

Reply:
I want answer this because it's clear that you just want to bother me 🤦🏻
I will not ignore your criticisms, if they are correct, I will correct the errors 
Ty

Reply:
Okay, I corrected the mistakes